src/lib/confmgt/var_type_def_st.h

```diff
@@ -151,17 +170,11 @@ struct var_type_def_t {
    * calling the functions in this type's function table.
    */
   const void *params;
-
-  /** True iff a variable of this type can never be set directly by name. */
-  bool is_unsettable;
-  /** True iff a variable of this type is always contained in another
-   * variable, and as such doesn't need to be dumped or copied
-   * independently. */
-  bool is_contained;
-  /** True iff a variable of this type can be set more than once without
-   * destroying older values. Such variables should implement "mark_fragile".
-   */
-  bool is_cumulative;
+  /**
+   * A bitwise OR of one or more VTFLAG_* values, describing properties
+   * for all values of this type.
+   **/
+  unsigned flags;
```

Comment:
Should we use a specific bit-width here?
Or will we never have more than 32 flags, so there's no reason to worry about different platforms having different widths?

Reply:
May as well use uint32_t; I'll make that change.

Reply:
Fixup in e8b7bc0057d063 with followup in 249f5c5fbafbf5

Reply:
.